cloudant-client/src/main/java/com/cloudant/client/api/Search.java

```diff
@@ -360,25 +360,26 @@ public Search counts(String[] countsfields) {
     }
 
     /**
-     * @param fieldName  the name of the field
-     * @param fieldValue the value of the field
+     * @param fieldName   the name of the field
+     * @param fieldValues field values to add
      * @return this for additional parameter setting or to query
      * @see <a target="_blank" href="https://docs.cloudant.com/search.html#faceting">drilldown
      * query parameter</a>
      */
-    public Search drillDown(String fieldName, String fieldValue) {
+    public Search drillDown(String fieldName, String... fieldValues) {

```

Comment:
I wrote a quick test and found that this will break API compatibility and code compiled against the old API will fail at runtime with a `java.lang.NoSuchMethodError` because the signature has changed.

You'll need to provide two overloads of the same method to ensure API compatibility.